astroquery/alma/tests/test_alma.py

```diff
@@ -464,7 +464,9 @@ def test_sia():
     sia_mock = Mock()
     empty_result = Table.read(os.path.join(DATA_DIR, 'alma-empty.txt'),
                               format='ascii')
-    sia_mock.search.return_value = Mock(table=empty_result)
+    mock_result = Mock()
```

Comment:
To be backwards compatible, this should work without any changes.

Reply:
Yeap, it will all be backwards incompatible as the whole point was to not return pyvo objects but tables.

Reply:
OTOH, I expect the only change users need to do is to drop their own `to_table`, at least that's what I see users were doing for the IRSA method. (Actually, they were doing `to_table().to_pandas()` most of the time)

Reply:
But it's a rather abrupt change. We could use the `enhanced_results` flag to sway them into that direction, first by introducing it, then giving them a deprecation warning before flipping the flag to on by default. It's less disruptive.